jbmc/regression/jbmc-inheritance/cprover_nondet_initialize/Test.java

```diff
@@ -0,0 +1,45 @@
+import org.cprover.CProver;
+
+class A {
+  int x;
+
+  public void cproverNondetInitialize() { CProver.assume(x == 42); }
+}
+
+class B extends A {}
+
+class C extends B {}
+
+class F extends A {
+  public void cproverNondetInitialize() {}
+}
+
+abstract class J {
+  int x;
+
+  public void cproverNondetInitialize() { CProver.assume(x == 42); }
+
+  abstract void neverCalled();
```

Comment:
:question: what is the purpose of this method?

Reply:
I thought an abstract class needed an abstract method. I'll remove it.